src/Equinox/Tracing.fs

```diff
@@ -0,0 +1,107 @@
+module Equinox.Core.Tracing
+
+open System.Diagnostics
+
+let source = new ActivitySource("Equinox")
+
+module Tags =
+    // General tags
+
+    /// The full stream name
+    [<Literal>]
+    let stream_name = "eqx.stream_name"
+    /// The ID of the stream
+    [<Literal>]
+    let stream_id = "eqx.stream_id"
+    /// The category of the stream
+    [<Literal>]
+    let category = "eqx.category"
+    /// The store being used
+    [<Literal>]
+    let store = "eqx.store"
+
+    // Information about loading events
+
+    /// The version of the stream at read time
+    [<Literal>]
+    let read_version = "eqx.read_version"
+    /// The configured batch size
+    [<Literal>]
+    let batch_size = "eqx.batch_size"
+    /// The total number of batches loaded from the store
+    [<Literal>]
+    let batches = "eqx.batches"
+    /// The total number of events loaded from the store
+    [<Literal>]
+    let loaded_count = "eqx.count"
+    /// The total bytes loaded during Transact/Query
+    [<Literal>]
+    let loaded_bytes = "eqx.bytes"
+    /// The version we load forwards from
+    [<Literal>]
+    let loaded_from_version = "eqx.load.from_version"
+    /// The load method (BatchForward, LatestEvent, BatchBackward, etc)
+    [<Literal>]
+    let load_method = "eqx.load.method"
+    /// Whether we needed to use the leader node
+    [<Literal>]
+    let requires_leader = "eqx.load.requires_leader"
+    /// Whether we used a cached value (regardless of whether we also reloaded)
+    [<Literal>]
+    let cache_hit = "eqx.cache.hit"
+    /// If we found a cached value, how long ago was it cached?
+    [<Literal>]
+    let cache_age = "eqx.cache.item_age_ms"
+    /// Whether we accept AnyCachedValue
```

Comment:
not sure if this is useful if there is a `eqx.cache.max_age_ms <> 0`

Reply:
Fair, we can derive the information 👍